feature-libs/organization/administration/core/store/effects/org-unit.effect.ts

```diff
@@ -384,11 +389,24 @@ export class OrgUnitEffects {
         this.orgUnitConnector
           .createAddress(payload.userId, payload.orgUnitId, payload.address)
           .pipe(
-            switchMap((data) => [
-              new OrgUnitActions.CreateAddressSuccess(data),
-              new OrgUnitActions.CreateAddressSuccess({ id: undefined }),
-              new OrganizationActions.OrganizationClearData(),
-            ]),
+            switchMap((data) => {
+              const successActions = [
+                new OrgUnitActions.CreateAddressSuccess(data),
+                new OrgUnitActions.CreateAddressSuccess({ id: undefined }),
+                new OrganizationActions.OrganizationClearData(),
+                new GlobalMessageActions.AddMessage({
```

Comment:
It would be better to find a way to listen for the effect to complete and then have the message and routing actions performed by that listener.